assembl/scripts/gen_chatbot_css.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh
```

Comment:
What is the script for?

Also, most things in Assembl's backend are bound to Python, and script execution to `fabric`. I'm not against `sh` scripts at all. But, for consistency, and for single execution style, this could be done in fabric quite easily. Convert the scss to `jinja2` template, and use a command similar to `docker_compose` task, to parse the jinja template with the appropriate data. OR, you can create this once in the themes folder. And parse it ad-hoc per discussion and load it via webpack.

Reply:
it's to generate the css you have to paste in the chatbot config page at motion.ai

Reply:
Is there a motionAI API to POST this content to their config?

Reply:
doesn't look like it https://docs.motion.ai/docs/api https://docs.motion.ai/v1.0/reference

Reply:
Dang, that sucks. Okay, I don't have anything against usage of shell script. But this script is not configurable enough to be used by all developers. I don't know how relevant this is. For now, this should be good enough to start. However, it needs to be documented. So, for that, I would request you add docs to this PR on usage.

Reply:
`botBackgroundColor='blue' botTextColor='white" ./gen_chatbot_css.sh` is too hard to use ?

Reply:
you only need to run it when you create a new bot at motion.ai

Reply:
For those not familiar with bash scripting, this may not be obvious you need to add variables before the command. For me it is. :)
You should document the usage of this script in a comment at the top of the script. And that for variables that are not defined, default values in this script are used.

Reply:
doc or comment then ?

Reply:
If you have a specific doc to create the chatbot, describe it in the doc then.